OpenRA.Game/Map/Map.cs

```diff
@@ -165,41 +183,65 @@ static object LoadVideos(MiniYaml y)
 			return videos;
 		}
 
-		[FieldLoader.Ignore] public Lazy<Dictionary<string, ActorReference>> Actors;
+		[FieldLoader.Ignore]
```

Comment:
I'm not a fan of positioning annotations and properties in the same line, even best practices tools like Resharper advice against this... but ohh well, if you wish so then so be it.